.github/workflows/scripts/generate-new-contributors.js

```diff
@@ -0,0 +1,95 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+import { simpleGit } from 'simple-git';
+const git = simpleGit();
+
+const REPO_NAME = "opentelemetry-collector-contrib"
+const REPO_OWNER = "open-telemetry"
+
+const firstTimeContributorText = "We are thrilled to welcome our new first-time contributors to this project. Thank you for your contributions "
```

Comment:
nit
```suggestion
const firstTimeContributorText = "We are thrilled to welcome new first-time contributors to this project. Thank you for your contributions "
```

Reply:
this doesn't really seem right to me? didn't you mean to remove the "new" instead of the "our" 🤔

Reply:
Rereading this morning, yes, I think you're correct